include/TPP/Passes.td

```diff
@@ -280,24 +280,34 @@ def Cleanup : Pass<"cleanup", "func::FuncOp"> {
   let summary = "General IR cleanup e.g., canonicalization, CSE etc.";
 }
 
-def LocalDialectsLowering : Pass<"lower-local-dialects", "ModuleOp"> {
-  let summary = "Lower all local dialects (XSMM, check etc.).";
+def LowLevelParallelization : Pass<"low-level-paralle", "ModuleOp"> {
+  let summary = "Low level parallelization (multi-threading, AMX config).";
   let dependentDialects = ["affine::AffineDialect",
                            "arith::ArithDialect",
                            "func::FuncDialect",
                            "memref::MemRefDialect",
-                           "check::CheckDialect",
-                           "perf::PerfDialect",
                            "scf::SCFDialect",
-                           "tensor::TensorDialect",
                            "xsmm::XsmmDialect",
                            "LLVM::LLVMDialect"];
   let options = [
     ListOption<"parallelTaskGrid", "parallel-task-grid",
            "unsigned", "Grid-sizes for parallel tasks.">
```

Comment:
nit: some default value like `1`

Reply:
I'm not sure the list option has a default value, none of the others do. Since this is just an NFC, I'll leave it for later.